pkg/shell/shell.html

```diff
@@ -1529,6 +1544,58 @@ <h4 class="modal-title" id="account-confirm-delete-title"></h4>
       </div>
     </div>
 
+    <script id="authorized-keys-tmpl" type="x-template/mustache">
+      {{#keys}}
+        <tr>
+        <td>{{comment}}</td>
+        <td class="fingerprint">
+           {{fp}}{{^valid}}<span translatable="yes">Invalid key</span>{{/valid}}
+        </td>
+        <td class="cell-buttons"><button data-raw="{{raw}}" class="btn btn-default accounts-self-privileged account-remove-key">-</button></td>
```

Comment:
It's destructive, but I feel it's not super-destructive, since you can just add the key back again and no data is really lost. Is there a potential bad outcome of just removing it?